README.md

```diff
@@ -16,5 +16,5 @@ Keybase needs to be [installed locally](https://keybase.io/download) since the s
 Your respective Twitter list can be gotten by ["requesting your data"](https://help.twitter.com/en/managing-your-account/how-to-download-your-twitter-archive) under your Twitter settings. Specifically, the `followers.j` and `following.js` files are what this script is designed to work with from the bundle that Twitter returns.
 
 ## To run
-Simply place your Twitter user file into the `data` folder and then run `run.py` using:
+Simply uncompress your downloaded Twitter personal data archive, and add only the files `followers.j` and `following.j` to the `data` folder, and then run `run.py` using:
```

Comment:
Thanks for the suggestion/PR!

There's just one correction with this to be done before we can merge: the files are `.js` files. Once that's fixed, this should be good to go.

```suggestion
Simply uncompress your downloaded Twitter personal data archive, add only the files `followers.js` and `following.js` to the `data` folder, and then run `run.py` using:
```